CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Fixed syncing exceptions to repeating events ([#641])
+
```

Comment:
What happened to the entries from the main branch? Also, each change should go in its respective section.